tests/test_bounds.sh

```diff
@@ -1,21 +1,19 @@
 #!/bin/sh
-# This script is a draft. Note: Bash is not available on CheriBSD.
-# This is a simple way to test the examples by checking ``stderr``.
+# This is a simple way to test "bounds.c" by checking ``stderr``.
+set -e
 
 : "${SSHPORT:=10021}"
-RESULT_riscv64="fail"
+RESULT="fail"
```

Comment:
Do we need to give this variable a default value?

Reply:
Not really. I'll fix that.